config/templates/cloud-config-controller

```diff
@@ -80,13 +80,14 @@ coreos:
         - name: 10-etcd.conf
           content: |
             [Unit]
-            Requires=cfn-etcd-environment.service
+            Wants=cfn-etcd-environment.service
             After=cfn-etcd-environment.service
 
             [Service]
-            EnvironmentFile=/etc/etcd-environment
+            EnvironmentFile=-/etc/etcd-environment
```

Comment:
Couldn't find the documentation for it but does the dash(-) is meant to instruct this unit to be started even when the environment file doesn't exist?